Makefile

```diff
@@ -76,7 +76,7 @@ integration: ## Run integration test
 
 .PHONY: test-e2e
 test-e2e: ## Run e2e test
-	go test -timeout 20m -v sigs.k8s.io/cluster-api-provider-aws/test/machines -kubeconfig $${KUBECONFIG:-~/.kube/config} -ssh-key $${SSH_PK:-~/.ssh/id_rsa} -cluster-id $${ENVIRONMENT_ID:-""} -ginkgo.v
+	go test -timeout 20m -v sigs.k8s.io/cluster-api-provider-aws/test/machines -kubeconfig $${KUBECONFIG:-~/.kube/config} -ssh-key $${SSH_PK:-~/.ssh/id_rsa} -actuator-image gcr.io/k8s-cluster-api/aws-machine-controller:0.0.1 -cluster-id $${ENVIRONMENT_ID:-""} -ginkgo.v
```

Comment:
Should we express the actuator image as an environment variable too? That way, if CI or anybody wants to test a different image we don't have to edit the file.